ReactiveTask/Task.swift

```diff
@@ -46,9 +46,22 @@ public struct Task {
 	}
 }
 
+private extension CollectionType where Generator.Element == String {
+	func escaped() -> [String] {
+		let whitespaceCharacterSet = NSCharacterSet.whitespaceCharacterSet()
+		return map { string -> String in
+			if string.rangeOfCharacterFromSet(whitespaceCharacterSet) != nil {
```

Comment:
I would extract this to a method in terms of `String`. I think it doesn't make sense to have the canonical implementation for this on a `CollectionType` extension.